### PR TITLE
set up the news backend

### PR DESCRIPTION
**Pull request by HGoyal09**
image

image

Set up the news data with a cron job 

image

### CODE REVIEW

File: data/market-news.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+const {stockNewsList} = require("../dal/stock-markets");
+const MARKET_NEWS = [{
+    country: 'USA',
+    news_sources: [
+        {
+            id: "Bloomberg",
```
Comment (weiyoung) [inline data/market-news.js]: inconsistency: should be called "bloomberg" in lower case. make sure to change in mongoDB too

File: dal/stock-markets.js [hunk 2]
```diff
@@ -1,10 +1,16 @@
 const mongoose = require("mongoose");
 
-mongoose.connect(process.env.STOCK_MARKET_MONGO_DB_CONNECTION_STRING, {
+// mongoose.connect(process.env.STOCK_MARKET_MONGO_DB_CONNECTION_STRING, {
+//     useNewUrlParser: true,
+//     useUnifiedTopology: true
+// });
+
+mongoose.connect('mongodb+srv://investobull:investobull9876!@stock-price-prediction.bg5zs.mongodb.net/StockMarket', {
```
Comment (weiyoung) [inline dal/stock-markets.js]: pls don't expose our api key publicly

**Review by weiyoung — COMMENTED** [review]
IMPORTANT!! fix api key part

**Review by paulsgh — APPROVED** [review]

**Review by weiyoung — APPROVED** [review]
LGTM

**Pull request merged**